scripts/patch-realsense-ubuntu-L4T.sh

```diff
@@ -52,62 +49,82 @@ fi
+#if [ "6.0" = "$PATCHES_REV" ]; then
+#    sudo apt install libdpkg-perl=1.21.1ubuntu2.1 -y
+#fi
 
+# Get the required tools to build the patched modules
+sudo apt-get install build-essential git libssl-dev -y
+if [ "6.0" != "$PATCHES_REV" ]; then
 # Get the linux kernel repo, extract the L4T tag
-echo -e "\e[32mRetrieve the corresponding L4T git tag the kernel source tree\e[0m"
-l4t_gh_dir=../linux-${KERNEL_RELEASE}-source-tree
-if [ ! -d ${l4t_gh_dir} ]; then
-	mkdir ${l4t_gh_dir}
+	echo -e "\e[32mRetrieve the corresponding L4T git tag the kernel source tree\e[0m"
+	l4t_gh_dir=../linux-${KERNEL_RELEASE}-source-tree
+	if [ ! -d ${l4t_gh_dir} ]; then
+		mkdir ${l4t_gh_dir}
+		pushd ${l4t_gh_dir}
+		git init
+		git remote add origin git://nv-tegra.nvidia.com/linux-${KERNEL_RELEASE}
+		# Use NVIDIA script instead to synchronize source tree and peripherals
+		#git clone git://nv-tegra.nvidia.com/linux-${KERNEL_RELEASE}
+		popd
+	else
+		echo -e "Directory ${l4t_gh_dir} is present, skipping initialization...\e[0m"
+	fi
+
+	#Search the repository for the tag that matches the maj.min for L4T
 	pushd ${l4t_gh_dir}
-	git init
-	git remote add origin git://nv-tegra.nvidia.com/linux-${KERNEL_RELEASE}
-	# Use Nvidia script instead to synchronize source tree and peripherals
-	#git clone git://nv-tegra.nvidia.com/linux-${KERNEL_RELEASE}
+	TEGRA_TAG=$(git ls-remote --tags origin | grep ${JETSON_L4T_VERSION} | grep '[^^{}]$' | tail -n 1 | awk -F/ '{print $NF}')
+	echo -e "\e[32mThe matching L4T source tree tag is \e[47m${TEGRA_TAG}\e[0m"
 	popd
-else
-	echo -e "Directory ${l4t_gh_dir} is present, skipping initialization...\e[0m"
 fi
 
-#Search the repository for the tag that matches the maj.min for L4T
-pushd ${l4t_gh_dir}
-TEGRA_TAG=$(git ls-remote --tags origin | grep ${JETSON_L4T_VERSION} | grep '[^^{}]$' | tail -n 1 | awk -F/ '{print $NF}')
-echo -e "\e[32mThe matching L4T source tree tag is \e[47m${TEGRA_TAG}\e[0m"
-popd
-
-
 #Retrieve tegra tag version for sync, required for get and sync kernel source with Jetson:
 #https://forums.developer.nvidia.com/t/r32-1-tx2-how-can-i-build-extra-module-in-the-tegra-device/72942/9
 #Download kernel and peripheral sources as the L4T github repo is not self-contained to build kernel modules
 sdk_dir=$(pwd)
-echo -e "\e[32mCreate the sandbox - NVidia L4T source tree(s)\e[0m"
+echo -e "\e[32mCreate the sandbox - NVIDIA L4T source tree(s)\e[0m"
 mkdir -p ${sdk_dir}/Tegra
 TEGRA_SOURCE_SYNC_SH="source_sync.sh"
 if [ "5.0.2" = "$PATCHES_REV" ]; then
-  TEGRA_SOURCE_SYNC_SH="source_sync_5.0.2.sh"
+	TEGRA_SOURCE_SYNC_SH="source_sync_5.0.2.sh"
+fi
+if [ "6.0" = "$PATCHES_REV" ]; then
+	TEGRA_SOURCE_SYNC_SH="source_sync_6.0.sh"
+	TEGRA_TAG="jetson_36.3"
 fi
 cp ./scripts/Tegra/$TEGRA_SOURCE_SYNC_SH ${sdk_dir}/Tegra
-#Download NVidia source, disregard errors on module tag sync
+#Download NVIDIA source, disregard errors on module tag sync
 ./Tegra/$TEGRA_SOURCE_SYNC_SH -k ${TEGRA_TAG} || true
```

Comment:
Please add this function and call as we did on the MIPI driver:
https://github.com/IntelRealSense/realsense_mipi_platform_driver/blob/c8549695060290340d8d9aceac2f5dbfd74830d6/setup_workspace.sh#L38

Reply:
This will will require thorough testing. Please open another ticket for that.

Reply:
NP